dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/AbstractMethodProblem.java

```diff
@@ -25,16 +29,12 @@
   MethodSymbol methodSymbol;
 
   AbstractMethodProblem(
-      ClassFile implementationClass,
-      MethodSymbol methodSymbol, ClassFile supertype) {
+      ClassFile implementationClass, MethodSymbol methodSymbol, ClassFile supertype) {
 
     // implementationClass is the source of the invalid symbolic reference, and supertype is the
     // target of the symbolic reference.
     super(
-        " does not exist in the implementing class",
-        implementationClass,
-        methodSymbol,
-        supertype);
+        " does not exist in the implementing class", implementationClass, methodSymbol, supertype);
```

Comment:
It's consistent that we have a space at the beginning and not at the end of the first argument. Something to look at later, not in this PR.